include/tc/autotuner/genetic_search.h

```diff
@@ -113,6 +115,7 @@ class GeneticSearch {
   Population population;
   TuningConfiguration lastBestConf;
   const size_t kMaxPopulationSize;
+  const size_t kMatingPoolSize;
```

Comment:
is this being exposed as an option to set? like we specify pop_size etc. via flags

Reply:
Right now it is not. I'll expose it in a subsequent commit.

Reply:
awesome :)